api/main.go

```diff
@@ -16,8 +16,10 @@ func Start() {
 	})
 
 	// Register all the routes and there corresponding handlers
-	router.GET("/api/projects", getAllOrgs)
-	router.GET("/api/projects/:id", getOrg)
+	router.GET("/api/orgs", getAllOrgs)
+	router.GET("/api/orgs/:id", getOrg)
+	router.GET("/api/orgs/:id/projects/:projectId", getProject)
+	router.GET("/api/orgs/:id/projects", getProjectByYear)
```

Comment:
Rename `getProjectByYear` to `getProjectsByYear`.

Reply:
Renamed in new commit.